routes/ssrf.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+const {
+  sinks: { ssrf }
+} = require('@contrast/test-bench-utils');
+
+const EXAMPLE_URL = 'http://www.example.com';
+
+module.exports = ({ router }) => {
+  router.get('/ssrf', (ctx) =>
+    ctx.render('../node_modules/@contrast/test-bench-content/views/ssrf', {
+      requestUrl: EXAMPLE_URL
+    })
+  );
+
+  const libs = ['axios', 'bent', 'fetch', 'request', 'superagent'];
+
+  libs.forEach((lib) => {
+    router.get(`/ssrf/${lib}/unsafe`, async (ctx, next) => {
+      const url = createUnsafeUrl(ctx.query.input);
+      const data = await makeRequest(lib, url);
+
+      ctx.body = data;
+    });
+
+    router.post(`/ssrf/${lib}/unsafe`, async (ctx, next) => {
+      const url = createUnsafeUrl(ctx.request.body.input);
+      const data = await makeRequest(lib, url);
+
+      ctx.body = data;
+    });
+
+    router.get(`/ssrf/${lib}/safe`, async (ctx, next) => {
+      const url = createSafeUrl(ctx.query.input);
```

Comment:
This needs to be `query['untrusted-input']` based on the key used in the UI.

Reply:
Isn't 'input' more idiomatic for our use?

Reply:
Yeah, but it's not what the `@contrast/test-bench-content` page is using. Either we should update that to send `input` or update what all the APIs do.

Reply:
I'd prefer to update the template unless you have serious concerns?

Reply:
I've updated the test-bench-content repo to use `name='input'` in the markup.